### ISSUE TITLE
Venkatakrishnan limiter ignores the periodic image, so it limits the wrong increment across a seam

### ISSUE THREAD

**Issue by ctdegroot** [DeGrootResearchGroup/aquaflux#143]
## The defect

`VenkatakrishnanLimiter.limit` (`aquaflux/schemes/limiter.py`) forms the unlimited increment as

```python
delta_minus = dot(gradient[cell], x_face - cell_geometry.centroid[cell])
```

where `cell` is `face_cells.owner` on one side and `face_cells.safe_neighbour` on the other. For the neighbour side this indexes the **raw** centroid.

`FaceCellConnectivity.neighbour_centroid` states the rule this breaks:

> Every displacement-forming operator (the owner→neighbour vector a diffusion, gradient, or Rhie–Chow term needs) must gather the neighbour centroid through *this* accessor rather than indexing `cell_centroid[safe_neighbour]` directly. On an ordinary interior or boundary face the two agree; across a periodic seam the raw neighbour centroid sits a full period away […]

So on a periodic seam the limiter's `x_face - centroid[neighbour]` spans a full domain length instead of the physical gap.

## Why it matters

The consumer already does this correctly. `LimitedUpwind.face_value` (`aquaflux/discretization/advection.py`) reconstructs with

```python
x_upwind = jnp.where(outflow[:, None], x_cell[fc.owner], fc.neighbour_centroid(x_cell))
reconstruction = phi_upwind + psi_upwind * dot(grad_upwind, face.centroid - x_upwind)
```

so across a seam **the limiter constrains a different increment from the one actually applied**. `psi` is computed for a displacement O(L) while the reconstruction uses one O(dx). The bounding property the limiter exists to provide is void there, in either direction (over- or under-limiting depending on the gradient's sign).

Nothing raises; the run just silently loses boundedness on seam cells.

## Suggested fix

Pass the *gathered per-face position* into the per-face helper instead of indexing inside it — `centroid[owner]` for the owner side and `face_cells.neighbour_centroid(centroid)` for the neighbour side — so the limiter forms the same displacement the scheme reconstructs on.

## Test gap

No test exercises a periodic mesh together with `LimitedUpwind`: the periodic cases (`test_periodic_channel`, `test_coupled_periodic_channel`, `test_coupled_mass_flow`, `test_channel_law_of_the_wall`) all use first-order advection, and the limited cases (`test_limited_advection`, `test_advection`) are non-periodic. A regression test should assert that on a periodic mesh the limiter's `delta_minus` for a seam face equals the reconstruction offset `LimitedUpwind` uses — i.e. that a smooth periodic field is *not* limited at the seam (`psi == 1`), which the current code will fail.

## Status

Found during a limiter audit on the pitzDaily study. **Latent** — pitzDaily is not periodic, so this is not implicated in any current result. Filed on its own merits.
